docker-compose.yml

```diff
@@ -34,7 +34,7 @@ services:
       - pktfwdr:/var/pktfwd
```

Comment:
Need to do the version number too

Reply:
Oops tagged wrong line, but lines 11 and 63

Reply:
Thanks